content/swiftui/concepts/gestures/gestures.md

```diff
@@ -0,0 +1,141 @@
+Description: 'Defines interactions from taps, clicks, and swipes to fine-grained gestures.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'iOS'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'SwiftUI ViewModifiers'
+  - 'SwiftUI'
+  - 'iOS'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+Views can respond to taps, drags, pinches and other gestures just by adding **Gesture** modifiers to your View.
+
+## Tap Gestures
+
+This is the most basic type of Gesture.
+The `.onTapGesture()` modifier can be called on the view desired to track the gesture on.
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onTapGesture {
+                print("View Tapped") 
+            }
+        
+    }
+}
+
+```
+
+The output looks like:
+
+![Tap_Gesture_SwiftUI.gif](https://raw.githubusercontent.com/Codecademy/docs/main/media/Tap_Gesture_SwiftUI.gif)
+
+Every time the text view "Hello, World" is tapped, it prints "View Tapped" in the console.
+The action can also be implemented after certain amount of tap gestures. For example:
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onTapGesture(count: 3) {
+                print("View Tapped") 
+            }
+        
+    }
+}
+```
+In this example it prints "View Tapped" after taping the text view three times.
+
+> **Note:** When a button is needed, use a Button instance rather than a tap gesture. Any view can be used as the button’s label, and the button type automatically provides many of the standard behaviors that users expect from a button, like accessibility labels and hints. 
+
+## Long Press Gestures
+
+`onLongPressGesture()` it's used for handling long presses gesture. `onLongPressGesture()` have an extra parameter called `minimumDuration:` (takes Double as input), it's used to specify after how much time the action will be executed. For example:
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onLongPressGesture(minimumDuration: 2) {
+                print("View Tapped")
+            }
+        
+    }
+}
+```
+The output looks like:
+
+![Tap_Gesture_SwiftUI.gif](https://raw.githubusercontent.com/Codecademy/docs/main/media/Long_Gesuter_SwiftUI.gif)

```

Comment:
Correct the spelling on the file name, and, use dashes instead of underscores, and make it all lower case.

Place some more descriptive text between the `[...]`.